coap-endpoint/src/command.rs

```diff
@@ -0,0 +1,53 @@
+//! Http context command handlers
+//!
+//! Contains actors that handles commands for HTTP endpoint
+
+use crate::error::CoapEndpointError;
+use coap_lite::{CoapOption, CoapRequest, CoapResponse, ResponseType};
+use drogue_cloud_endpoint_common::commands::Commands;
+use drogue_cloud_service_common::Id;
+
+use actix_rt::time::timeout;
+use std::collections::LinkedList;
+use std::net::SocketAddr;
+use std::time::Duration;
+
+const HEADER_COMMAND: CoapOption = CoapOption::Unknown(4210);
```

Comment:
Please move these option number constants to a single location, and add a comment describing the reason for introducing it.